secrets/secrets-kubernetes/main.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+
+	"github.com/nats-io/nats.go"
+	"github.com/nats-io/nkeys"
+	"github.com/wasmCloud/wasmCloud-contrib/secrets/secrets-kubernetes/pkg/secrets"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	clientcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	ServiceName = "kube"
+)
+
+type kubeSecretsServer struct{}
+
+type kubeApplicationPolicy struct {
+	Impersonate string `json:"impersonate"`
+	Namespace   string `json:"namespace"`
+}
+
+func (s *kubeSecretsServer) Locator(name string) (string, string, error) {
+	parts := strings.Split(name, "/")
+	if len(parts) != 2 {
+		return "", "", secrets.ErrInvalidRequest
+	}
+
+	return parts[0], parts[1], nil
+}
+
+func parseApplicationPolicy(r *secrets.Request) (*kubeApplicationPolicy, error) {
+	rawPolicy, err := r.Context.Application.PolicyProperties()
+	if err != nil {
+		return nil, err
+	}
+	policy := &kubeApplicationPolicy{Namespace: "default"}
+	err = json.Unmarshal(rawPolicy, policy)
+	return policy, err
+}
+
+func (s *kubeSecretsServer) Get(ctx context.Context, r *secrets.Request) (*secrets.Response, error) {
+	policy, err := parseApplicationPolicy(r)
+	if err != nil {
+		return nil, fmt.Errorf("%w: invalid policy format", secrets.ErrInvalidRequest)
+	}
+	slog.Info("Get", slog.Any("application", r.Context.Application.Name), slog.Any("key", r.Name), slog.Any("impersonate", policy.Impersonate))
```

Comment:
Nit, shouldn't these `slog.Any` calls mostly be `slog.String`?

Reply:
you're right! updating